.vscode/tasks.json

```diff
@@ -0,0 +1,38 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            
+            "label": "Create venv with all exporters",
+            "type": "shell",
+            "linux": {
+                "command": "${workspaceFolder}/tools/prepare_virtual_env.sh ; ${workspaceFolder}/venv/Scripts/activate ; pip install -r requirements.txt",
+            },
+            "windows": {
+                "command": "@()|${workspaceFolder}/tools/prepare_virtual_env.bat ; ${workspaceFolder}/venv/Scripts/Activate.ps1 ; pip install -r requirements.txt",
+            }
+        },
+        {
+            "label": "Build UI",
+            "type": "shell",
+            "windows": {
+                "command": "${workspaceFolder}/venv/Scripts/Activate.ps1 ; python ${workspaceFolder}/setup.py build_ui"
+            },
+            "linux": {
+                "command": "${workspaceFolder}/venv/Scripts/activate ; python ${workspaceFolder}/setup.py build_ui"
+            }
+        },
+        {
+            "label": "Run all tests on all cores",
+            "type": "shell",
+            "windows": {
+                "command": "${workspaceFolder}/venv/Scripts/Activate.ps1 ; python -m pytest -n auto ${workspaceFolder}/test"
```

Comment:
you don't need to activate the venv. you can just use the python that is in the venv folder to do what you need

Reply:
Actually, we should use pull python from the VSCode extension. So it'd just be `{command:python.interpreterPath}`

Reply:
Sounds good. Then I can also remove the linux / windows specific sections except for creating the venv.